projects/common/camera.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#define GLM_ENABLE_EXPERIMENTAL
```

Comment:
Did you want to put this define in both camera.h and config.h?

Reply:
Yeah, for #reasons they have to exist in both places.